modules/values.tmpl.yaml

```diff
@@ -503,6 +503,7 @@ traefik:
         regex: apps.${base_domain}
         replacement: apps.${cluster_name}.${base_domain}
   dashboard:
+    enabled: true
```

Comment:
I prefer false by default

```suggestion
    enabled: false
```

Reply:
I made that to avoid breaking that for people who use it 
But indeed I would prefer a false too

Reply:
No problem, i will do the change for my platforms.
So false by default

Reply:
Change done :smile: